### PR TITLE
[IMPROVED][STOOD-62] - Add new param to filter task by taskStatus

### PR DESCRIPTION
**Pull request by wirazeta**
# Changelog

- Set param for filter task by status

## Related JIRA Card

- [STOOD-62](https://glorion.atlassian.net/browse/STOOD-62)

## The changes contain

- [x] new features
- [ ] bugfix
- [ ] code improvements
- [ ] hotfix
- [ ] breaking changes

### CODE REVIEW

File: docs/sql/temp.sql
Comment (adiatma85) [inline docs/sql/temp.sql:1]: Remove this file

File: src/business/entity/task.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ type Task struct {
 	CategoryID null.Int64  `db:"fk_category_id" json:"categoryId"`
 	Title      string      `db:"title" json:"title"`
 	Priority   int64       `db:"priority" json:"priority"`
-	TaskStatus string      `db:"task_status" json:"taskStatus"`
-	Periodic   string      `db:"periodic" json:"periodic"`
+	TaskStatus string      `db:"task_status" json:"taskStatus"`	//todo, ongoing, done
```
Comment (adiatma85) [inline src/business/entity/task.go]: Modify it to ``Enums(todo, ongoing, done)``
Do the similliar to Periodic field too

File: src/handler/task.go [hunk 2]
```diff
@@ -138,3 +139,5 @@ func (r *rest) DeleteTask(ctx *gin.Context) {
 
 	r.httpRespSuccess(ctx, codes.CodeSuccess, nil, nil)
 }
+
+// Filter task by taskStatus
```
Comment (adiatma85) [inline src/handler/task.go]: Remove these lines

File: docs/sql/temp.sql
Comment (wirazeta) [reply docs/sql/temp.sql:1]: done

File: src/business/entity/task.go (hunk 1 above)
Comment (wirazeta) [reply src/business/entity/task.go]: done

File: src/handler/task.go (hunk 2 above)
Comment (wirazeta) [reply src/handler/task.go]: done

File: src/handler/task.go [hunk 3]
```diff
@@ -137,4 +138,4 @@ func (r *rest) DeleteTask(ctx *gin.Context) {
 	}
 
 	r.httpRespSuccess(ctx, codes.CodeSuccess, nil, nil)
-}
+}
```
Comment (adiatma85) [inline src/handler/task.go]: Add blankspace in here

**Review by adiatma85 — APPROVED** [review]
LGTM

**Pull request merged**